sympy/calculus/util.py

```diff
@@ -61,15 +62,18 @@ def continuous_domain(f, symbol, domain):
     try:
         sings = S.EmptySet
         if f.has(Abs):
-            sings = solveset(1/f, symbol, domain)
+            sings = solveset(1/f, symbol, domain) + \
+                solveset(denom(together(f)), symbol, domain)
         else:
             for atom in f.atoms(Pow):
-                predicate, denom = _has_rational_power(atom, symbol)
-                if predicate and denom == 2:
-                    sings = solveset(1/f, symbol, domain)
+                predicate, denomin = _has_rational_power(atom, symbol)
+                if predicate and denomin == 2:
+                    sings = solveset(1/f, symbol, domain) +\
+                        solveset(denom(together(f)), symbol, domain)
                     break
             else:
-                sings = Intersection(solveset(1/f, symbol), domain)
+                sings = Intersection(solveset(1/f, symbol), domain) + \
+                    solveset(denom(together(f)), symbol, domain)
```

Comment:
Does this 2nd part need to be `Intersection(solveset(denom(together(f)), symbol), domain)`, too?

Reply:
The current code seems to work correctly, though I am not very sure why the intersection is not needed in the second part. The reason, I think might be that the need to take an intersection is arising only when the function is a negative power of a trignometric function. While solving for `denom(together(f))`, there is no negative power, as any negative powers in the denominator are sent to the numerator. For example:
```
>>>f = 1/(1/sin(x) + 1/cos(x))
>>> denom(together(f))
sin(x) + cos(x)
```